### ISSUE TITLE
Missing Apple Pay native payment confirmation

### ISSUE THREAD

**Issue by ayushin** [jonasbark/flutter_stripe_payment#46]
Great job implementing native payments in Objective-C!

What is missing is the functionality to call back iOS and tell it wether the native payment was successful (see for example https://github.com/rbii-incorporated/stripe_native `StripeNative.confirmPayment` without which the Apple Pay just keeps spinning until it times out.

I am totally new to Objective-C (and honestly got too much on my plate right now to learn a new framework) but I've done a stub with https://github.com/jonasbark/flutter_stripe_payment/pull/45) perhaps somebody could finish it (or re-implement) completely and I'll test it?

Basically when a native payment is completed (https://developer.apple.com/documentation/passkit/pkpaymentauthorizationcontrollerdelegate/2867952-paymentauthorizationcontroller?language=objc) a handler is provided to call back with the result.

Can somebody implement it or give me some pointers?

**Comment by ayushin** [comment]
OK, so this now works in master for Apple Pay. Great job!
